test/toolchains/BUILD.bazel

```diff
@@ -0,0 +1,77 @@
+load("//scala:scala_toolchain.bzl", "scala_toolchain")
+
+scala_toolchain(
+    name = "ast_plus_one_deps_unused_deps_warn_impl",
+    dependency_mode = "plus-one",
+    dependency_tracking_method = "ast",
+    unused_dependency_checker_mode = "warn",
+    visibility = ["//visibility:public"],
+)
+
+toolchain(
+    name = "ast_plus_one_deps_unused_deps_warn",
```

Comment:
is this used? Can't see it and I think maybe unused deps warn isn't tested
I think this [line](https://github.com/bazelbuild/rules_scala/pull/995/files#diff-23f6e936cb498d27c3ef44340314fd82R53) was supposed to be a different test that used it but was forgotten

Reply:
Yes it was supposed to be used in test_plus_one_ast_analyzer_unused_deps